Microsoft.Toolkit.Uwp.SampleApp/readme.md

```diff
@@ -35,9 +35,10 @@ Here is an example:
 ```
 
 You can define "interactive" values in this file. The value types can be:
-* String: You want the user to provide a text. The string is built like this @[Name:**String**:Default value]
+* String: You want the user to provide a text. The string is built like this @[Name:**String**:Default value:UpdateSourceTriggerId]
```

Comment:
What's this new piece here, it's not called out more, and I didn't notice a code change for the string type?

Reply:
Removed it.